src/material/core/_core.scss

```diff
@@ -7,6 +7,12 @@
 @use './focus-indicators/focus-indicators';
 @use './typography/all-typography';
 
+// Private sass variables that will be used as reference throughout component stylesheets
+$private-default-focus-indicator-border-width: 3px;
+$private-default-focus-indicator-border-style: solid;
+$private-default-focus-indicator-border-color: black;
```

Comment:
I think the default here should be `transparent`, not `black`, as by default focus indicators shouldn't show unless in HCM. Borders that are `transparent` should render in HCM.

Reply:
Gotcha, sounds good. Thank you!